app/src/components/layout/navLinks/NavLinks.tsx

```diff
@@ -8,25 +8,74 @@ import { useSessionContext } from '../../../providers/sessionProvider/SessionPro
 const NavLinks = () => {
     const navigate = useNavigate();
     const [session] = useSessionContext();
-
+    const [width, setWidth] = useState<number>(window.innerWidth);
     const nav = (e: ReactEvent<HTMLAnchorElement, MouseEvent>, link: string) => {
         e.preventDefault();
         navigate(link);
     };
 
+    console.log(width);
+    useEffect(() => {
+        const handleResizeWindow = () => setWidth(window.innerWidth);
+        window.addEventListener('resize', handleResizeWindow);
+        return () => {
+            window.removeEventListener('resize', handleResizeWindow);
+        };
+    }, []);
+
     return session.isLoggedIn ? (
-        <Header.Nav>
-            <Header.NavItem role="link" className="clickable" onClick={(e) => nav(e, routes.HOME)}>
-                Home
-            </Header.NavItem>
+        <Header.Nav style={{ display: 'initial', color: 'white' }}>
             <Header.NavItem
+                style={{ color: 'white' }}
                 role="link"
                 className="clickable"
-                data-testid="logout-btn"
-                onClick={(e) => nav(e, routes.LOGOUT)}
+                onClick={(e) => nav(e, routes.START)}
             >
-                Log Out
+                Home
             </Header.NavItem>
+            {width <= 990 ? (
+                <>
+                    <Header.NavItem
+                        role="link"
+                        className="clickable"
+                        style={{ color: 'white' }}
```

Comment:
When focused, this text will become white on yellow background which is poor in colour contrast.
Maybe better to make a css class to override style, so that we can config &:focus and possible other stuffs in one place